synapse/tests/test_tools_cryo_cat.py

```diff
@@ -37,8 +37,9 @@ def test_cryocat(self):
 
                 outp = self.getTestOutp()
                 argv = ['--list', addr]
-                self.eq(s_cryocat.main(argv, outp), 1)
-                self.true(outp.expect('Currently requires --authfile'))
+                with self.assertLogs(logger='synapse.tools.cryo.cat', level='ERROR') as logs:
```

Comment:
I'd use our `getLoggerStream()` helper here over assertLogs since we can just check the contents of stream vs. a specific message via index

Reply:
+1 `getLoggerStream`

Reply:
ok

Reply:
:+1: